src/components/cards/DarkModeCard.vue

```diff
@@ -0,0 +1,72 @@
+<template>
```

Comment:
Not right now, but at some point in the future, I think we should update this card to work with arbitrary themes. I'm sure you were already thinking of doing things like holiday themes, and this card could probably be reused to prompt the user to switch to that theme 😃